assets/training/finetune_acft_common/components/validation/spec.yaml

```diff
@@ -0,0 +1,171 @@
+$schema: https://azuremlschemas.azureedge.net/latest/commandComponent.schema.json
+type: command
+
+version: 0.0.1
+name: finetune_common_validation
+display_name: Common Validation Component
+description: Component to validate the finetune job against Validation Service
+
+is_deterministic: True
+
+environment: azureml://registries/azureml/environments/acpt-pytorch-2.0-cuda11.7/versions/21
+
+code: ../../src/validation
+
+inputs:
+
+  # component input: mlflow model path
+  mlflow_model_path:
+    type: mlflow_model
+    optional: true
+    description: MLflow model asset path. Special characters like \ and ' are invalid in the parameter value.
+
+  # ###################################### Data validation ###################################### #
+  # component input: training mltable
+  training_data:
+    type: mltable
+    optional: false
+    description: Path to the mltable of the training dataset.
+
+  # optional component input: validation mltable
+  validation_data:
+    type: mltable
+    optional: true
+    description: Path to the mltable of the validation dataset.
+
+  # component input: test mltable
+  test_data:
+    type: mltable
+    optional: true
+    description: Path to the mltable of the test dataset.
+
+  # ###################################### Compute validation ###################################### #
+  compute_preprocess:
+    type: string
+    optional: true
+    description: compute to be used for preprocess eg. provide 'FT-Cluster' if your compute is named 'FT-Cluster'. Special characters like \ and ' are invalid in the parameter value. If compute cluster name is provided, instance_type field will be ignored and the respective cluster will be used.
```

Comment:
**C**ompute 
(C in capital) and same at other places

Reply:
fixed, Thanks